src/main/java/com/ibm/g11n/pipeline/client/IndustryDomain.java

```diff
@@ -0,0 +1,98 @@
+/*  
+ * Copyright IBM Corp. 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ibm.g11n.pipeline.client;
+
+/**
+ * Industry domain constants used by translation request data.
+ * 
+ * @author yoshito_umaoka
+ */
+public enum IndustryDomain {
+    /**
+     * Aerospace and the military-industrial complex
+     */
+    AEROMIL("Aerospace and the military-industrial complex"),
```

Comment:
No APIs are available for fetching available list and its localized form at this point.
I'm not sure it's worth adding a dedicated API just for the keyword. Although we may add "generic" style API for accessing available enum values, including others later.